src/main/java/uk/gov/companieshouse/api/strikeoffobjections/common/ApiLogger.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.companieshouse.api.strikeoffobjections.common;
```

Comment:
would uk.gov.companieshouse.api.strike_off_objections.common be better for the package name
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Reply:
Agreed. Follows standard so will change.

Reply:
The CH standard seems to be what you have so leave as is